windows/sound.cpp

```diff
@@ -560,13 +560,10 @@ CSound::CSound ( void           (*fpNewCallback) ( CVector<int16_t>& psData, voi
     // in case we do not have a driver available, throw error
     if ( lNumDevs == 0 )
     {
-        throw CGenErr ( "<b>" + tr ( "No ASIO audio device (driver) found." ) + "</b><br><br>" +
-            tr ( "The " ) + APP_NAME + tr ( " software requires the low latency audio "
-            "interface ASIO to work properly. This is not a standard "
-            "Windows audio interface and therefore a special audio driver is "
-            "required. Either your sound card has a native ASIO driver (which "
-            "is recommended) or you might want to use alternative drivers like "
-            "the ASIO4All driver." ) );
+        throw CGenErr ( "<b>" + tr ( "No audio device with ASIO driver detected" ) + "</b><br><br>" +
+            tr ( "To use " ) + APP_NAME + tr ( ", you need to install an ASIO driver first. "

```

Comment:
I wonder if this construct makes translations harder. Previously, it was just an article ("The"), now it's an infinitive construction. This works for German, but I'm not sure about other languages.
The following may be more flexible, but I'm not sure if it's better overall. Maybe someone else can comment as well. I think @pljones once mentioned this pattern.

`QString ( tr ( "The %1 software requires..." ) ).arg ( APP_NAME )`

(Hrm, I'm pretty sure I had already typed this once, but somehow it got lost!?)

Reply:
Ah yes - this format does make it clearer for the translators.  I'd forgotten about it, too!  This way, they get more of the context, it's not so broken up.